libs/sdk-backend-spi/src/workspace/dashboards/utils.ts

```diff
@@ -0,0 +1,131 @@
+// (C) 2019-2021 GoodData Corporation
+import noop from "lodash/noop";
+import {
+    IDashboardLayout,
+    IDashboardLayoutItem,
+    IDashboardLayoutSection,
+    isDashboardLayout,
+    DashboardWidget,
+} from "./layout";
+import { IWidget, IWidgetDefinition, isWidget, isWidgetDefinition } from "./widget";
+
+/**
+ * Has dashboard layout only empty sections and widgets?
+ * @alpha
+ */
+export const isDashboardLayoutEmpty = (layout: IDashboardLayout<any>): boolean => {
+    return layout.sections.every((section) => section.items.length === 0);
+};
+
+/**
+ * Represents nested path in layout
+ * It's useful to track the layout location of the widget
+ * @alpha
+ */
+export type LayoutPath = Array<string | number>;
```

Comment:
Can you please extend the docs with what the string a number mean? Number is presumably index, but the string?